src/sample-account-setup.test.ts

```diff
@@ -1,27 +1,546 @@
+          {
+            given: ['Alice'],
+            family: 'Smith',
+          },
+        ],
+        photo: [
+          {
+            contentType: 'image/png',
+            url: 'https://docs.medplum.com/img/cdc-femaledoc.png',
+          },
+        ],
+      },
+      'Practitioner/practitioner'
+  );
+
+  patient.generalPractitioner = [createReference(practitioner)];
+  await medplum.updateResource(patient);
+
+  //Create a schedule for the practitioner
+  const schedule = await medplum.createResourceIfNoneExist<Schedule>(
+      {
+        resourceType: 'Schedule',
+        id: 'schedule',
+        actor: [
+          {
+            reference: createReference(practitioner).reference,
+            display: getDisplayString(practitioner),
+          },
+        ],
+      },
+      'Schedule/schedule'
+  );
+
+  //Populate the schedule with slots
+  const slots: Slot[] = [];
+  const slotDate = new Date();
+  for (let day = 0; day < 60; day++) {
+    for (const hour of [9, 10, 11, 13, 14, 15]) {
+      slotDate.setHours(hour, 0, 0, 0);
+      const slot = await medplum.createResourceIfNoneExist<Slot>(
+          {
+            resourceType: 'Slot',
+            id: `slot-${day}-${hour}`,
+            start: slotDate.toISOString(),
+            schedule: createReference(schedule),
+          },
+          'Slot/slot-' + day + '-' + hour
+      );
+      slots.push(slot);
+    }
+    slotDate.setDate(date.getDate() + 1);
+  }
+
+  //Create observations for DiagnosticReport if not exits
+  const hemoglobinA1c = await medplum.createResourceIfNoneExist<Observation>(
+      {
+        resourceType: 'Observation',
+        id: 'hemoglobinA1c',
+        subject,
+        code: {
+          text: 'Hemoglobin A1c',
+        },
+        valueQuantity: {
+          value: 5.4,
+          unit: 'mmol/L',
+        },
+        referenceRange: [
+          {
+            high: {
+              value: 7.0,
+            },
+          },
+        ],
+      },
+      `Observation/hemoglobinA1c`
+  );
+
+  //All of these resources are created in the same transaction if they don't exist
+
+  //Add CarePlan - create 2, 1 active and one completed
+  const activeCarePlan = await medplum.createResourceIfNoneExist<CarePlan>(
+      {
+        resourceType: 'CarePlan',
+        status: 'active',
+        intent: 'order',
+        id: 'active-care-plan',
+        activity: [
+          {
+            detail: {
+              code: {
+                text: 'Antenatal education',
+              },
+              location: {
+                display: 'FOOMEDICAL HOSPITAL AND MEDICAL CENTERS'
+              },
+              status: 'in-progress'
+            }
+          },
+        ],
+        title: 'Routine antenatal care',
+        period: {
+          start: date.toISOString(),
+        },
+        category: [
+          {
+            text: 'Routine antenatal care',
+          }
+        ],
+        subject,
+      },
+      'CarePlan/active-care-plan'
```

Comment:
Same issue. This also needs to be related to patient. I'm not sure the best way to handle this, but will ask for guidance from @codyebberson and @rahul1 - namely this should be something like `CarePlan\patient.id+antenatal.  Also, @rahul1 - we need to upgrade this to something more in line with our recommendations.